extensions/sql/policy-store-sql/src/main/java/org/eclipse/dataspaceconnector/sql/policy/store/schema/postgres/PostgresDialectStatements.java

```diff
@@ -38,16 +38,16 @@ public String getFormatAsJsonOperator() {
 
     @Override
     public SqlQueryStatement createQuery(QuerySpec querySpec) {
-        if (querySpec.containsAnyLeftOperand("prohibitions")) {
+        if (querySpec.containsAnyLeftOperand("policy.prohibitions")) {
```

Comment:
since these statements are referring to the policy, it's necessary to have it made explicit in the left operand?
if I think about a query to search for policies probably I wouldn't think about putting the `policy` at the beginning of the operand. Same for the others, I'm not sure about the need for the root object name.

Reply:
Unfortunately prepending the `policy.` is necessary, because otherwise it wouldn't conform to the `PolicyDefinition.java` class anymore. Then, we'd have to add some special hack here, to make the reflective access work again, which is a bad idea IMO.